docs/intro.md

```diff
@@ -23,7 +23,9 @@ This allows your end users' browsers to cache the (fairly weighty) Kiln applicat
 
 ## Usage
 
-As Kiln itself is a component, it must be included in your layouts and have some data, e.g. `allow: true` (a convention we use for components that don't otherwise have data in them). Add an instance of Kiln to your bootstraps:
+As Kiln itself is a component, it must be included in your layouts and have some data, e.g. `allow: true` (a convention we use for components that don't otherwise have data in them). 
```

Comment:
```suggestion
As Kiln itself is a component, it must be included in your layouts and have some data, e.g. `allow: true` (a convention we use for components that don't, otherwise have data in them). 
```